tools/dev-upload.sh

```diff
@@ -0,0 +1,74 @@
+#!/bin/bash
+
+set -e
+
+if [ -z "${RELEASE}" ]; then
+  echo "ERROR: RELEASE environment variable needs to be set to the" >&2
+  echo "  target Ubuntu version." >&2
+  exit 1
+fi
+
+MIR_VERSION=$( dpkg-parsechangelog --show-field Version \
+               | perl -n -e '/(^[^~-]+)/ && print $1' )
+GIT_REVISION=$( git rev-parse --short HEAD )
+
+source <( python <<EOF
```

Comment:
I'd welcome a better way to achieve this (pass multiple values out from a python script), as this will pass through if the python script fails for some reason.

I could print `ERROR=0` at the end to make sure the script completed, or not print anything until it did… Suggestions welcome.

_Moved from a previous commit_